File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        solutions = []
        for i in range(len(nums)):
            if nums[i] in solutions:
                index = solutions.index(nums[i])
                return [index, i]
            solutions.append(target - nums[i])
            print(solutions)
        return []

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int i = 0;
        int j = nums.length - 1;
        while(i < nums.length){
            if (i == nums.length - 1){
                break;
            }
            if(nums[i] == nums[j]){
                return true;
            } else {
                if(j - 1 == i){
                    j = nums.length - 1;
                    i++;
                } else {
                    j--;
                }
            }
        }
        return false;
    }
}
